app/views/users/_sidebar.html.erb

```diff
@@ -48,34 +48,21 @@
       <div class="widget">
         <div class="daily-stats">
           <h4>Daily Stats</h4>
-          <p>3 Visits</p>
-          <p>35 Minutes on Pages</p>
-          <p>4 Articles Read</p>
-          <p>1456 Words Read</p>
+          <p id="visits-per-day"><%= pluralize @user.number_visits_per_day, "Visit" %> </p>
+          <p id="average-active-time-per-day"><%= pluralize @user.average_active_time_per_day, "Second" %>Spent Reading</p>
+          <p id="average-number-articles-per-day"><%= pluralize @user.number_articles_per_day, "Article" %> Read</p>
+          <p id="average-number-words-per-day"><%= pluralize @user.number_words_per_day, "Word" %> Read </p>
```

Comment:
replaced the placeholders by the actual data and pluralization of words

Reply:
Perfect. Thanks for doing this!